### ISSUE TITLE
Unable to allow access for the app to be included in a page

### ISSUE THREAD

**Issue by gteti** [aquariophilie/axolotl#29]
I've installed the project on teti.atlassian.net and tried adding it to a page: https://teti.atlassian.net/wiki/spaces/MS/pages/5701646/Copy+of+Test+QR-Code+app

It says I should grant access which I do but the system doesn't move forward.
![immagine](https://user-images.githubusercontent.com/51110452/132955995-a2c29ffe-3dff-4f5e-a279-645cbc529e91.png)

Possible ideas and fixs ?

**Comment by gmacario** [comment]
If not earlier, let us review this when we meet on 2021-09-16

**Comment by Raffone17** [comment]
You can share apps in development as written in the following guide [https://developer.atlassian.com/platform/forge/manage-your-apps/](https://developer.atlassian.com/platform/forge/manage-your-apps/):

- go to the console ([https://developer.atlassian.com/console/myapps/](https://developer.atlassian.com/console/myapps/) )
- select the app and  
- go to the distribution tab
- click on edit button
- change the sharing status.

These are the required fields for sharing: Company/Department* , Customer support contact* and Privacy policy* .

If you want to distribute the app you can follow the guide [https://developer.atlassian.com/platform/forge/distribute-your-apps/](https://developer.atlassian.com/platform/forge/distribute-your-apps/).

**Comment by gteti** [comment]
We have found out that the "Allow Access" on my home PC was probably blocked by the Pop-UP blocker extension. Thus rendering the process incomplete. If it is completed you can start to see the application in your website. I now have another issue while @GGn0 was able to deploy it's own version to his website.

**Comment by gmacario** [comment]
Let me check whether I understood it correctly:

1. @GGn0 managed to install the app on https://scalzone.atlassian.net/ (is this the correct URL?) following the instructions at #35
2. @gteti still has issues to install the app on https://gteti.atlassian.net/

@GGn0, @gteti: Is my understanding correct?

If so I would merge #35 while I would keep #29 and reassign it to @gteti

**Comment by gteti** [comment]
> Let me check whether I understood it correctly:
> 
> 1. @GGn0 managed to install the app on https://scalzone.atlassian.net/ (is this the correct URL?) following the instructions at [[WIP] [docs] Add install documentation #35](https://github.com/aquariophilie/axolotl/pull/35)
> 2. @gteti still has issues to install the app on https://gteti.atlassian.net/
> 
> @GGn0, @gteti: Is my understanding correct?
> 
> If so I would merge #35 while I would keep #29 and reassign it to @gteti

I was able to install it too but have some other issue which I can't check before Monday evening.

**Comment by gmacario** [comment]
OK, thanks @gteti for confirming.

Let us discuss installation instructions in related PR #35

**Comment by gmacario** [comment]
@gteti 

> I was able to install it too but have some other issue which I can't check before Monday evening.

Did you eventually manage to check the other issues you were facing?

**Comment by gteti** [comment]
I managed to went further and gave access to the app. I can't go on since I get:
![immagine](https://user-images.githubusercontent.com/51110452/134405578-d4900ac8-9a90-4962-9a66-a4a70375f4d9.png)

I even tried: 

> You can share apps in development as written in the following guide https://developer.atlassian.com/platform/forge/manage-your-apps/:
>
> -    go to the console (https://developer.atlassian.com/console/myapps/ )
> -    select the app and
> -   go to the distribution tab
> -    click on edit button
> -    change the sharing status.
>
>These are the required fields for sharing: Company/Department* , Customer support contact* and Privacy policy* .
>
> If you want to distribute the app you can follow the guide https://developer.atlassian.com/platform/forge/distribute-your-apps/.
